rotkehlchen/api/v1/schemas.py

```diff
@@ -1713,8 +1713,6 @@ def _validate_single_oracle_id(
 
 
 class GenericNonTokenAssetSchema(Schema):
-    # TODO: When adding custom assets we need to check if this schema needs to be expanded
-    # or we need a different one
```

Comment:
For this todo we have a new schema. Does the name make sense now? Is one of the things we discussed and I believe is misleading now. Since non token asset could be a custom asset

Reply:
Makes sense, renamed to `CryptoAssetSchema`

Reply:
But is not a crypto asset since tokens don't use this schema afaik

Reply:
How would you name it then?

Reply:
That's why the TODO was there :sweat_smile: and the question was `Does the name make sense now?`.  Maybe `BasicCryptoAsset` is a good name and checking it I'm seeing that it shares logic and fields with `EvmTokenSchema` so maybe we can create something that is the base for both like `BasicCryptoSchema` and then have `EvmTokenSchema` and `SimpleAssetSchema`. What do you think?